tests/benchmark_overhead_works.rs

```diff
@@ -0,0 +1,69 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use assert_cmd::cargo::cargo_bin;
+use std::{process::Command, result::Result};
+use tempfile::tempdir;
+
+static RUNTIMES: [&'static str; 6] = ["polkadot", "kusama", "westend", "rococo", "wococo", "versi"];
+
+/// `benchmark-overhead` works for all dev runtimes.
+#[test]
+#[cfg(unix)]
+fn benchmark_overhead_works() {
+	for runtime in RUNTIMES {
+		assert!(benchmark_overhead(format!("{}-dev", runtime)).is_ok());
+	}
+}
+
+/// `benchmark-overhead` rejects all non-dev runtimes.
+#[test]
+#[cfg(unix)]
```

Comment:
Why does this only works on Unix?

Reply:
All the other tests in that directory have it as well, don't know why.